derive_builder_core/src/builder.rs

```diff
@@ -294,11 +295,9 @@ impl<'a> ToTokens for Builder<'a> {
                     }
                 ));
 
-                if self.std {
-                    tokens.append_all(quote!(
-                        impl std::error::Error for #builder_error_ident {}
```

Comment:
In ≥1.81, is there some blanket impl on `std::error::Error` that means all `core::error::Error` impls are satisfactory? If not, do we need to emit both impls to avoid breaking compatibility?